website/docs/ide-config.md

```diff
@@ -0,0 +1,62 @@
+# Vscode
+
+> Vscode Extensions
+
+- ESLint
+  - id: "dbaeumer.vscode-eslint"
+  - recommend reason: "Integrates ESLint JavaScript into VS Code."
+- Stylelint
+  - id: "stylelint.vscode-stylelint"
+  - recommend reason: "Official Stylelint extension for Visual Studio Code"
+- GitLens
+  - id: "eamodio.gitlens"
+  - recommend reason: "It helps you to visualize code authorship at a glance via Git blame annotations and CodeLens"
+- markdownlit
+  - id: "DavidAnson.vscode-markdownlint"
+  - recommend reason: "Markdown linting and style checking for Visual Studio Code"
+
+> Vscode Settings
+
+```json
```

Comment:
you can use `js` or `json5` to ignore red highlighting
![image](https://user-images.githubusercontent.com/48400568/169688436-bcd1bc49-ed06-4de7-b496-274d3f30a9a9.png)